Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjerciciooUno
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            
+            // se instancia pedro             
+            Pedro opedro = new Pedro ();
+
+            // se crea modelo persona de pedro
+            ModeloPersona modelopedro = new ModeloPersona ("Blanco",45,"pedro perez");
+
+            //se llena info de pedro
+            opedro.llenarInfo (modelopedro);
```

Comment:
Delete space between method and parameters 


 `opedro.llenarInfo(modelopedro);`

Reply:
has been corrected